document/CoreFormat.mdk

```diff
@@ -1445,6 +1453,36 @@ or if the tool cannot or chooses not to determine whether the result occurred at
 or if the tool examines source code written in a language that lacks such a construct, the `suppressionStates` array
 shall not include the value `"suppressedInSource"`.
 
+### `baselineState` property { #result-baselineState }
+
+If the `baselineId` property (&sect;[#run-baselineId]) of the current `run` object (&sect;[#run]) is present,
+then the `result` object may contain a property named `baselineState` whose value is a string that shall specify
+the state of this `result` with respect to the run specified by `baselineId`.
+
+If the `run.baselineId` property is absent, the `baselineState` property shall be absent.
+
+This property shall have one of the following values, with the specified meanings:
+
+* `"new"`: This result was detected in the current run but was not detected in the run specified by `run.baselineId`.
+
+* `"existing"`: This result was detected both in the current run and in the run specified by `run.baselineId`.
+
+* `"absent"`: This result was detected in the run specified by `run.baselineId` but was not detected in the current run.
+
+If the `run.baselineId` property is present but the `baselineState` property is absent, the `baselineState` property
+shall be considered to have the value `"new"`.
+
+~ IsoNote
+The purpose of the `baselineState` property is to allow result management systems to keep track of how many
+new results are introduced in each run, and how many have been addressed.
```

Comment:
Ended up with "The purpose of the `baselineState` property is to allow (for example) a measurement of how many new results were introduced in the run, and how many previously existing results no longer appear." The "for example" is because you can imagine more complicated scenarios involving multiple runs, like trend lines.

---
In reply to: [63958732](https://github.com/sarif-standard/sarif-spec/pull/203#discussion_r63958732) [](ancestors = 63958732,63957749,63952480)